src/core/pyproxy.c

```diff
@@ -1225,9 +1232,69 @@ EM_JS_NUM(int, pyproxy_init_js, (), {
 });
 // clang-format on
 
+JsRef
+create_once_proxy(PyObject* obj)
+{
+  Py_INCREF(obj);
+  return (JsRef)EM_ASM_INT(
+    {
+      let o = { $$ : { ptr : $0 } };
```

Comment:
Remind me what's "$$" in this context?

Reply:
This is a hack, I've fixed it in #1340, I suppose I will copy the code from that branch back to this one.